app/Models/DatabaseDAO.php

```diff
@@ -259,15 +272,18 @@ public function dbCopy(string $filename, int $mode, bool $clearFirst = false): b
 					$error = 'Error: SQLite import file is not readable: ' . $filename;
 				} elseif ($clearFirst) {
 					$userDAO->deleteUser();
+					$userDAO = FreshRSS_Factory::createUserDao();
```

Comment:
Fix regression from https://github.com/FreshRSS/FreshRSS/pull/6285 for some cases